src/components/mobile-icons/MobileIcon.jsx

```diff
@@ -0,0 +1,77 @@
+// @flow strict
+
+import React from 'react';
+import classNames from 'classnames';
+import type {IconColorType} from '../icons/Icon';
+
+export type IconTypeType =
+  | 'answer-bubble'
+  | 'ask-bubble'
+  | 'clear'
+  | 'comment'
+  | 'crown'
+  | 'crown-outline'
+  | 'cup'
+  | 'flashlight-off'
+  | 'flashlight-on'
+  | 'heart-outline'
+  | 'keyboard'
+  | 'options'
+  | 'sent'
+  | 'share-on-ios'
+  | 'star-outline';
+
+export type SizeType = 'small' | 'medium' | 'normal';
+
+export const TYPE: {[name: string]: IconTypeType, ...} = {
```

Comment:
Not sure if that's possible in Flow, but in typescript you could create union type from either object key or value. So in case you add new value, type is updating automatically

Reply:
It's also possible but in Flow union type from object values like `{a: 'small', b: 'big'}` will be `string` not a union `'small' | 'big'`

Reply:
ok I see, that's a bummer :(